control/setup_control.go

```diff
@@ -468,3 +474,9 @@ func UpdateTrapzBlock(ctx context.Context, name string, maxVel float64, dependsO
 	}
 	return nil
 }
+
+// TunedPIDErr returns an error with the stored tuned PID values.
+func TunedPIDErr(tunedVals PIDConfig) string {
+	return fmt.Sprintf("{\"p\": %v, \"i\": %v, \"d\": %v, \"type\": \"%v\"}",
+		tunedVals.P, tunedVals.I, tunedVals.D, tunedVals.Type)
+}
```

Comment:
Why not return an error directly instead of a string?

```suggestion
func TunedPIDErr(tunedVals PIDConfig) string {
	return fmt.Errorf("{\"p\": %v, \"i\": %v, \"d\": %v, \"type\": \"%v\"}",
		tunedVals.P, tunedVals.I, tunedVals.D, tunedVals.Type)
}
```